homeassistant/components/google_generative_ai_conversation/__init__.py

```diff
@@ -139,7 +139,17 @@ async def async_setup_entry(
     """Set up Google Generative AI Conversation from a config entry."""
 
     try:
-        client = genai.Client(api_key=entry.data[CONF_API_KEY])
+        client: Client | None = None
+
+        def init_client():
+            nonlocal client
+            client = Client(api_key=entry.data[CONF_API_KEY])
+
+        await hass.async_add_executor_job(init_client)
+
+        if client is None:
+            raise HomeAssistantError("Google Generative AI client is not initialized")
```

Comment:
I don't see how client could be None

Reply:
I was a mypy thing, since it was none, it could still be none I guess.

Reply:
It doesn't pass mypy check otherwise